streampark-console/streampark-console-webapp/src/views/flink/home/components/Modal.vue

```diff
@@ -100,13 +100,13 @@
       flinkHome: formValue.flinkHome,
     });
     const checkResp = parseInt(resp.data);
-    if (checkResp != 0) {
+    if (checkResp !== FlinkEvnEnum.FEASIBLE) {
```

Comment:
Using a switch case would be better. What do you think? 
```
if (checkResp !== FlinkEvnEnum.FEASIBLE) {
      switch (checkResp) {
        case FlinkEvnEnum.INVALID:
          Swal.fire('Failed', 'FLINK_HOME invalid path.', 'error');
          break;
        case FlinkEvnEnum.NAME_REPEATED:
          Swal.fire('Failed', t('setting.flinkHome.operateMessage.flinkNameIsUnique'), 'error');
          break;
        case FlinkEvnEnum.FLINK_DIST_REPEATED:
          Swal.fire(
            'Failed',
            'can no found flink-dist or found multiple flink-dist, FLINK_HOME error.',
            'error',
          );
      }
      changeOkLoading(false);
      return;
    }
```

Reply:
sure